minikube/deploy_bactopia_pipeline.yaml

```diff
@@ -0,0 +1,82 @@
+# TODO - export NXF_ANSI_LOG="false" - Minimise resources
+# TODO - export NXF_EXECUTOR='local' - Just for the bactopia process
```

Comment:
These env vars are already exported below.
Also `NXF_EXECUTOR='local'` is set in bactopia process, not here.

Reply:
Removed TODOs